ui/admin/app/routes/scopes/scope/roles/role/manage-scopes/manage-org-projects.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class ScopesScopeRolesRoleManageScopesManageOrgProjectsRoute extends Route {
+  // =attributes
+
+  queryParams = {
+    search: {
+      refreshModel: true,
+      replace: true,
+    },
+    page: {
+      refreshModel: true,
+    },
+    pageSize: {
+      refreshModel: true,
+    },
+  };
+
+  // =services
+
+  @service store;
+
+  // =methods
+
+  /**
+   * Loads projects for current org scope.
+   * @return {Promise<{role: RoleModel, orgScope: [ScopeModel], projectScopes: [ScopeModel], totalItems: number, totalItemsCount: number, selectedProjectIDs: [string], remainingProjectIDs: [string]}> }
+   */
+  async model({ org_id, search, page, pageSize }) {
+    const role = this.modelFor('scopes.scope.roles.role');
+    const orgScope = this.store.peekRecord('scope', org_id);
+
+    let selectedProjectIDs = role.grantScopeProjectIDs;
+    let remainingProjectIDs = [];
+    if (role.scope.isGlobal) {
+      const projects = await Promise.all(
+        role.grantScopeProjectIDs.map((id) =>
+          this.store.findRecord('scope', id),
+        ),
+      );
+
+      selectedProjectIDs = projects.reduce(function (filtered, proj) {
```

Comment:
```suggestion
      selectedProjectIDs = projects.reduce((filtered, proj) => {
```

Also I wonder if `filtered` is the right word, I got confused initially because I thought that meant "filtered out" which actually was the `remainingProjectIDs`

Reply:
yeah i agree that is confusing, ill refactor this, great point